packages/augur-ui/src/modules/common/form.styles.less

```diff
@@ -1168,6 +1167,7 @@ div.FormDropdownActive {
 
     .SingleDatePicker_picker {
       margin-top: -1px;
+      z-index: @above-all-content;
```

Comment:
Why mess with z-index?

Reply:
The two date picker dropdowns were overlapping eachother